### DISCUSSION TITLE
Assistance Required for GitHub Education Renewal

### DISCUSSION THREAD

**Discussion by mendax0110** [community/community#134735] [GitHub Education]
### Hi everyone,

<img width="469" alt="Rejection" src="https://github.com/user-attachments/assets/d53cb92b-00e6-4f4c-a02a-088fbf8f4af3">

I am currently still enrolled at the same institution, the Higher Technical Federal Training and Research Institute (HTL Rankweil), where I successfully received GitHub Education benefits last year (2022). However, I am encountering the following error message during the renewal process:

less

GitHub Education benefits are currently only available for accredited degree or diploma granting schools.

Your code school, boot camp, or other organized informal learning institution may apply to become a GitHub Campus Partner:
https://education.github.com/partners/schools

If approved, program participants will become eligible to receive all of the benefits of GitHub Education. 

I have verified my school-issued email address (adrian.goessl@student.htl-rankweil.at), which is correctly associated with my institution. Despite this, I am unable to proceed with the renewal.

Could you please assist me in resolving this issue? I am still an active student at HTL Rankweil, and I would greatly appreciate any help you can provide to renew my GitHub Education benefits.

Thank you for your time and assistance.

Best regards,

Adrian Goessl

**Comment by officialprosingh** [comment]
Here are a few steps you can take to resolve this issue:

Confirm Accreditation Status: Ensure that HTL Rankweil is listed as an accredited degree or diploma-granting institution. You might want to provide official documentation or a link to your school's accreditation status on its website.

Contact GitHub Support: Reach out to GitHub Education support directly. Explain your situation clearly, as you did in your post, and provide evidence of your enrollment and your institution’s accreditation. You can contact them through the GitHub Education support page or email.

Use School-Issued Email: Make sure you're using your verified school-issued email address (adrian.goessl@student.htl-rankweil.at) when applying for the renewal. This helps in verifying your student status.

Provide Additional Proof: If needed, provide additional proof of your enrollment, such as a current student ID, a letter from your school, or a recent transcript. This can help expedite the verification process.

Check GitHub Campus Program: If your institution is not already a GitHub Campus Partner, suggest that your school apply to become one. This can provide benefits to all students at your institution.

You can also check the [GitHub Education documentation](https://education.github.com/discount_requests/pack_application) for more detailed instructions on the application and renewal process.

I hope these steps help you renew your benefits. Good luck!

**Reply by mendax0110** [reply]
Thank you, i created a Ticket. :)
